ToyBox/classes/MainUI/MulticlassPicker.cs

```diff
@@ -111,7 +111,7 @@ public class MulticlassPicker {
                     if (!canSelectClass)
                         UI.Label("to select this class you must unselect at least one of your other existing classes".localize().orange());
                     if (optionsHasClass && chArchetype != null && archetypeOptions.Empty()) {
-                        UI.Label("due to existing archetype, ".localize() + $"{chArchetype.Name.yellow()}" + ",  this multiclass option will only be applied during respec.".localize().orange());
+                        UI.Label("due to existing archetype".localize() + ", " + $"{chArchetype.Name.yellow()}" + ", " + "this multiclass option will only be applied during respec.".localize().orange());
```

Comment:
This is not the right way to localize a sentence with commas.  It should get converted to a localized format string.  Please fix this in a follow up patch.